hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -722,11 +722,11 @@
     </description>
   </property>
   <property>
-    <name>ozone.scm.chunk.layout</name>
+    <name>ozone.scm.container.layout</name>
```

Comment:
Changing config property names needs to be backward compatible by linking the old and new ones:

https://github.com/apache/ozone/blob/d09cdd4a585b66e95f32c8a054ee493101ee4de0/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java#L299-L309

Reply:
Thank you for letting me know this, I have addressed this issue in my latest commit.